streaming/index.js

```diff
@@ -122,21 +139,21 @@ const startWorker = async (workerId) => {
 
   const pgConfigs = {
     development: {
-      user:     process.env.DB_USER || pg.defaults.user,
```

Comment:
Wow. I did _not_ mean to reformat all of streaming/index.js. Backing all of that out.

Reply:
![force push](https://user-images.githubusercontent.com/517302/202028184-92cbea25-a817-4076-b484-870cd5a50e0a.gif) ![jedi mind trick like it never happened](https://user-images.githubusercontent.com/517302/202028420-633d1f66-46c5-429a-bba1-a2f697358878.gif)